vertical-pod-autoscaler/pkg/recommender/routines/recommender.go

```diff
@@ -36,6 +37,7 @@ import (
 var (
 	checkpointsWriteTimeout = flag.Duration("checkpoints-timeout", time.Minute, `Timeout for writing checkpoints since the start of the recommender's main loop`)
 	minCheckpointsPerRun    = flag.Int("min-checkpoints", 10, "Minimum number of checkpoints to write per recommender's main loop")
+	hoursThreshold          = flag.Int("hours-threshold", 1, "Number of hours before VPA object annotations are considered stale")
```

Comment:
The name of the parameter should be more descriptive and the unit should be smaller e.g. seconds because right now the smallest value is 1h and we might decide to decrease it in the future.
Perhaps a better name would be "freshness-update-interval-seconds".